util/portal-poster/README.md

```diff
@@ -0,0 +1,52 @@
+# Portal Poster
+
+## Purpose
+Zenhub user story [17978 - Ability to resend multiple reports through ReportStream's report API](https://app.zenhub.com/workspaces/reportstream-om-67feaaa15ecf34000f4cc206/issues/gh/cdcgov/prime-reportstream/17978) articulates the need to have an approach to re-process a cohort of SimpleReport CSV files based on a separate CSV file of identifiers used as input.  This project addresses the part of the process that posts the files to ReportStream.
```

Comment:
nit: might note that this is currently hardcoded to work with CP csv files, but can be modified to run batches of hl7 or fhir files.